### PR TITLE
Unit Scaling

### PR DESCRIPTION
**Pull request by PragTob**
Superseds #17 

Used the more general `unit_scaling` configuration option and put in a bunch of cleanup work leftover from the scenarios work. Made `units` a separate argument as well as units are the same for all scenarios of a group being displayed.

Thanks to original author @elpikel

### CODE REVIEW

**Comment by PragTob** [comment]
Annoyingly dialyzer fails on travis but passes with my local versions on travis --> upgrade versions to see if it's better (maybe all of elixir 1.4 creates weird errors?)

**Comment by PragTob** [comment]
still 3 builds failing. Only one build passing that is not 1.5.2 and runs dialyzer... Theory: PLT doesn't get updated because the benchee version is the same but the struct has changed and dialyzer doesn't care about git hashes. Now to find a way to force upgrade :)

**Comment by PragTob** [comment]
ok dialyxir is a good citizen and reads the whole deps.lock and hashes it... unless ther's something I'm missing this isn't the reason :|

**Comment by PragTob** [comment]
for better or worse, this seems to have worked but the damn excoveralls upload failed again... grrrmlll

**Comment by PragTob** [comment]
rebased/squashed to get rid off those commits where I tried out CI stuff

**Comment by PragTob** [comment]
OMG IT PASSES :tada:

File: mix.exs [hunk 1]
```diff
@@ -45,7 +45,7 @@ defmodule BencheeHTML.Mixfile do
   defp deps do
     [
       {:benchee,        github: "PragTob/benchee", branch: "master"},
-      {:benchee_json,   github: "devonestes/benchee_json", branch: "updating-to-scenarios"},
+      {:benchee_json,   github: "PragTob/benchee_json", branch: "master"},
```
Comment (devonestes) [inline mix.exs]: Ughhh, thanks for finally fixing this!

**Comment by PragTob** [comment]
Hm good questions, standard `units` code resides in benchee now, I might add some though. Thanks for the ping about that! :)

**Comment by PragTob** [comment]
and here are the test :grin:

**Pull request merged**